practical-react-unit-testing-with-jest-mocks-and-spies/Exercise1/solution/src/Select.test.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * @jest-environment jsdom
+ */
+//manually mocking toSelectOptions
+
+import { render } from "@testing-library/react";
+import { Select } from "./Select";
+import { toSelectOptions } from "./toSelectOptions";
+
+jest.mock("./toSelectOptions", () => ({
+  toSelectOptions: jest.fn().mockReturnValue([
+    { label: "2021", value: "2021" },
+    { label: "2022", value: "2022" },
+    { label: "2023", value: "2023" },
+  ]),
+}));
+describe("Year", () => {
+  test("Year works with one-off mocked values", () => {
+    toSelectOptions.mockImplementationOnce(() => {
+      return [{ label: "2024", value: "2024" }]
+    })
+
+    const result = render(
```

Comment:
Take a look at this exercise in CodeSandbox and you’ll find some linting warnings like:

> `result` is not a recommended name for `render` returned value. Instead, you should destructure it, or name it using one of: `view`, or `utils`eslint(testing-library/render-result-naming-convention)

Reply:
I got some other related warnings and am now just using `screen.getByText` instead of capturing the render result